Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Image/GetAllImageTests/GetAllImageTests.cs

```diff
@@ -113,12 +113,12 @@ private List<ImageDTO> GetImagesDTOList()
             };
         }
 
-        private void MockRepositoryAndMapper(List<Image> ImageList, List<ImageDTO> ImageListDTO)
+        private void MockRepositoryAndMapper(List<DAL.Entities.Media.Images.Image> ImageList, List<ImageDTO> ImageListDTO)
         {
             _mockRepo.Setup(r => r.ImageRepository.GetAllAsync(
-            It.IsAny<Expression<Func<Image, bool>>>(),
-            It.IsAny<Func<IQueryable<Image>,
-            IIncludableQueryable<Image, object>>>()))
+            It.IsAny<Expression<Func<DAL.Entities.Media.Images.Image, bool>>>(),
+            It.IsAny<Func<IQueryable<DAL.Entities.Media.Images.Image>,
+            IIncludableQueryable<DAL.Entities.Media.Images.Image, object>>>()))

```

Comment:
We can create entity at the head of file like that: using ImageEntity = DAL.Entities.Media.Images.Image
And then just use ImageEntity